### ISSUE TITLE
✅ 🐞 Elementor Pro v3.5.0 - Increase file size of frontend-lite.min.css

### ISSUE THREAD

**Issue by astbarc** [elementor/elementor#16852]
## Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

## Steps to reproduce
Version 3.5.0 of elementor pro increase the size of the file assets/css/frontend-lite.min.css.

The file size is more than double that of the previous version (66Kb). I think there is a problem with it. 

## Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a default WordPress theme active.

## Environment

<details>
<summary>System Info</summary>
```
<!-- For bug reports, let us know about your system environment: Copy and paste the system info report from Elementor => System info, and paste it here or in http://pastebin.com/ -->
```
</details>

**Comment by shilo-ey** [comment]
Thank you for reporting!

We will fix this soon and release a patch.

Nice catch

**Comment by sermalefico** [comment]
and eicons and fontawesome is allways loaded even is font awesome experiment is active

**Comment by rotemee** [comment]
@sermalefico can you please share a link to a page where this happens?

**Comment by astbarc** [comment]
Hi @shilo-ey 

There were 2 new versions of Elementor Pro and this issue has not been fixed yet.

Do you have an ETA for this?

Thanks.

**Comment by nicholaszein** [comment]
Hello!

This issue was solved with the release of Elementor Pro 3.6.0, a long time ago. Apologies for not closing it sooner! 🙏 

Kind regards
